lib/tree-view.coffee

```diff
@@ -53,7 +53,34 @@ class TreeView extends View
     @updateRoots(state.directoryExpansionStates)
     @selectEntry(@roots[0])
 
-    @selectEntryForPath(state.selectedPath) if state.selectedPath
+    # Now that directory expansion is done asyncly combining 2 different
```

Comment:
I wish we could figure out a way to rewrite tree-view incrementally without making it a giant project...